src/Parser.js

```diff
@@ -674,35 +682,81 @@ export default class Parser {
         }
     }
 
+    consumeComment() {
+        // the newline character is normalized in Lexer, check original source
+        while (this.nextToken.text !== "EOF" && this.nextToken.loc &&
+                this.nextToken.loc.getSource().indexOf("\n") === -1) {
```

Comment:
Why not `this.nextToken.text.indexOf("\n") === -1`?

Reply:
@kevinbarabash Lexer normalizes whitespaces to a single space(` `). I think it's better to look at the source, instead of separating whitespace token and newline token.

Reply:
That makes sense.  It might be worth adding a comment here about that.